scripts/process-image-manifests

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
+
+# This script extracts the most recent manifests from an image containing a manifests bundle and processes it into a version that works with this operator.
+#
+# Example: Running the script and checking the changes produced using git status
+#
+#$ ./process-image-manifests registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata:1.11.0-2
+#$ git status
+# Changes not staged for commit:
+#  (use "git add <file>..." to update what will be committed)
+#  (use "git checkout -- <file>..." to discard changes in working directory)
+#
+#	modified:   manifests/integreatly-3scale/3scale.package.yaml
+#
+#Untracked files:
+#  (use "git add <file>..." to include in what will be committed)
+#
+#	manifests/integreatly-3scale/3scale-0.5.0/
+
+
+set -e
+
+WORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+MANIFESTS_DIR=${WORK_DIR}/../manifests
+MANIFEST_TMP_DIR=${MANIFESTS_DIR}/temp
+IMAGE=$1
+FOLDER_PRODUCT_NAME=3scale
+
+function ver { printf "%03d%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
+
+cleanup (){
+    echo "Cleaning up ${MANIFEST_TMP_DIR}"
+    rm -rf ${MANIFEST_TMP_DIR}
+}
+
+trap 'cleanup' EXIT
+
+extract_manifests_from_image() {
+    echo "Creating ${MANIFEST_TMP_DIR} directory"
+    mkdir -p ${MANIFEST_TMP_DIR}
+    echo "Extracting manifests from image '${IMAGE}' to ${MANIFEST_TMP_DIR}"
+    oc image extract $IMAGE --path /manifests/:${MANIFEST_TMP_DIR} --confirm
+    ls -la ${MANIFEST_TMP_DIR}
+}
+
+get_current_csv() {
+    echo "Getting current csv  from existing package.yaml"
+    CURRENT_CSV=$(yq r ${MANIFESTS_DIR}/integreatly-${FOLDER_PRODUCT_NAME}/3scale.package.yaml channels[0].currentCSV | sed -e s/^3scale-operator.v//)
```

Comment:
If we are using yq, can we add some instructions on how to install it. Might be worth just adding it to the CI Dockerfile so we can at least run these scripts from that container if we decide we want to do that.